e2e/tests/conceptsets/copy-to-workspace.spec.ts

```diff
@@ -41,6 +35,8 @@ async function createConceptSet(
 }
 
 describe('Copy Concept Set to another workspace', () => {
+  const destWorkspace = 'e2eTestCopyConceptSetToWorkspace'; // The copy-to workspace with default CDR version
```

Comment:
The destination workspace (created with default CDR version name) doesn't need to be created new per test run. Reusing the same workspace reduces number of new workspaces per run and makes this test run faster.